cmd/helpers/mssql/mssql.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package mssql
+
+import (
+	"fmt"
+	"github.com/microsoft/go-sqlcmd/cmd/sqlconfig"
+	"github.com/microsoft/go-sqlcmd/pkg/sqlcmd"
+	"os"
+)
+
+func Connect(endpoint sqlconfig.Endpoint, user sqlconfig.User, console sqlcmd.Console) *sqlcmd.Sqlcmd {
+	// BUG(stuartpa): Check what the value of fromEnvironment should be here
+	v := sqlcmd.InitializeVariables(false)
+	s := sqlcmd.New(console, "", v)
+	s.Format = sqlcmd.NewSQLCmdDefaultFormatter(false)
+	connect := sqlcmd.ConnectSettings{
+		ServerName: fmt.Sprintf(
+			"%s,%d",
+			endpoint.EndpointDetails.Address,
+			endpoint.EndpointDetails.Port),
+		UseTrustedConnection: false,
```

Comment:
>seTrustedConnection: false, [](http://example.com/codeflow?start=3&length=27)

eventually user.UserDetails has to support integrated and AAD auth types, correct?

Reply:
Yes, completely agree.  I presume with docker installs no-one uses a trusted connection, hence the hardcode here.  But you raise a good point that this impl is focused on sql auth only at the moment, the 99.99% case I believe for docker.

Reply:
I've added authentication-type to sqlconfig, and this PR works for basic and trusted.  Added an issue for AAD auth, which I'll impl straight after this PR